src/Streamly/Internal/Data/Stream/IsStream/Top.hs

```diff
@@ -325,9 +343,9 @@ mergeInnerJoin = undefined
 --
 -- Time: O(m x n)
 --
--- /Unimplemented/
+-- /Pre-release/
```

Comment:
If I remember correctly, this function had some issue because of which it was marked as unimplemented. Did you review and test it? Does it always produce correct output? @adithyaov do you remember the issue since you reviewed it?

Reply:
Let's keep it unimplemented unless we have a correct implementation.